src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.intake;
+
+import static edu.wpi.first.units.Units.Radians;
+<<<<<<< HEAD
+=======
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+>>>>>>> b38babce7c2115f5fefb513ee61988edc8bcbf20
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+
+public class IntakeConstants {
+    public static final double ANGLE_RATIO = 0;
+<<<<<<< HEAD
+    public static MutableMeasure<Angle> DOWN_ANGLE = MutableMeasure.zero(Radians);
+    public static MutableMeasure<Angle> UP_ANGLE = MutableMeasure.zero(Radians);
+=======
+    public static final MutableMeasure<Angle> DOWN_ANGLE = MutableMeasure.zero(Radians);
+    public static final MutableMeasure<Angle> UP_ANGLE = MutableMeasure.zero(Radians);
+>>>>>>> b38babce7c2115f5fefb513ee61988edc8bcbf20
+
+    public enum IntakePose {
+        UP(IntakeConstants.UP_ANGLE),
+        DOWN(IntakeConstants.DOWN_ANGLE);
+        MutableMeasure<Angle> intakePose;
```

Comment:
Should be final

Reply:
OK.

Reply:
381418fb60abfd384af6a39eaa907c7483a6ecf4